site/content/languages/tinygo.md

```diff
@@ -153,10 +153,10 @@ target wasi.
   * Host calls the built-in export `malloc` to get the memory offset to write
     the string, which is passed as a parameter to the exported Guest function.
     The host owns that allocation, so must call the built-in export `free` when
-    done. The Guest uses `stringToPtr` to retrieve the string from the Wasm
+    done. The Guest uses `ptrToString` to retrieve the string from the Wasm
     parameters.
 * Guest passes a string to an exported Host function
-  * Guest uses `ptrToString` to get the memory offset needed by the Host
+  * Guest uses `stringToPtr` to get the memory offset needed by the Host
```

Comment:
another thing that may be backwards; should we also say "imported" host function? @codefromthecrypt

Reply:
yeah imported is more clear